client/src/scenes/CharacterSelect.lua

```diff
@@ -213,6 +213,10 @@ function CharacterSelect:createStageCarousel(player, width)
     stageCarousel:setPassengerById(player.settings.selectedStageId)
   end
 
+  stageCarousel.onPassengerUpdateCallback = function(carousel, selectedPassenger)
+    -- Specifically don't set to cause tons of loads
+  end
```

Comment:
I'm unsure if this isn't just causing more problems for an issue that is not truly urgent. You can for example enter the selection with keyboard and then ready up with mouse; in that case the player choice is not maintained. Same thing vice versa.